src/components/users/UserList.vue

```diff
@@ -45,18 +71,39 @@ export default {
   mounted() {
     this.$store.dispatch('application/setHeaderInfo', {
       title: 'Users',
-      subtitle: '',
+      subtitle: 'List',
```

Comment:
Subtitle should not show the number of items?

Reply:
It will be fixed in the header refactor.